src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx

```diff
@@ -42,6 +96,20 @@ For <InlinePopover type="apm" /> and infrastructure data to be integrated, all o
 
 If the integration is not working, see [Troubleshooting the APM-Infrastructure integration](/docs/infrastructure/new-relic-infrastructure/troubleshooting/apm-data-missing-infrastructure).
 
+## View logs for your APM and infrastructure data [#logs-context]
+
+You can also bring your logs and application's data together to make troubleshooting easier and faster. With [logs in context](/docs/logs/logs-context/configure-logs-context-apm-agents/), you can see log messages related to your errors and traces directly in your app's UI. You can also see logs in context of your [infrastructure data](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/), such as Kubernetes clusters. No need to switch to another UI page.
```

Comment:
"no need to switch to another UI page" is one of the benefits of using logs in context, and i think the value prop for the whole doc is being able to synthesize as many data sources as possible without leaving a single UI page